### DISCUSSION TITLE
Not starting after updating

### DISCUSSION THREAD

**Discussion by niklask52t** [dani-garcia/vaultwarden#4188] [Q&A]
### Subject of the issue
Not starting after updating

### Deployment environment

I have the container with following konfiguration 
```
docker run -d --restart=always --name vaultwarden \
  -e DOMAIN=https://DOMAIN \
  -e SMTP_HOST=smtp.HOST.MAIL \
  -e SMTP_FROM=mail@MAIL.MAIL \
  -e SMTP_PORT=465 \
  -e SMTP_SSL=false \
  -e SMTP_EXPLICIT_TLS=true \
  -e SMTP_USERNAME=mail@MAIL.MAIL \
  -e SMTP_PASSWORD=PASSWORD\
  -e SIGNUPS_ALLOWED=false \
  -e SIGNUPS_VERIFY=true \
  -e TRASH_AUTO_DELETE_DAYS=7 \
  -v /DATAPATH/:/data/ \
  -p IP:PORT \
  vaultwarden/server:latest
```

it was running fine, but everytime i try to update it, the container is not starting after and keep restarting

* vaultwarden version: 2023.5.0 

* Install method: Docker

**Comment by BlackDex** [comment]
Try to start without the `-d` to see what is going on.
Also, running v2023.5.0 as in a new version is probably not right, since Vaultwarden it self is on v1.30.1 which is using v2023.10.x as a web-vault version.

**Comment by niklask52t** [comment]
Thank you for your answer. In web ive using Version 2023.5.0. How can i see my main Vaultwarden version?

**Reply by BlackDex** [reply]
Go to `/admin/diagnostics`
